.gitignore

```diff
@@ -19,3 +19,5 @@ dist/*
 # you can add these to your own "global" .gitignore file.
 # https://help.github.com/articles/ignoring-files/#create-a-global-gitignore
 #
+.idea/
+.python-version
```

Comment:
These can go in your "global" `.gitignore` file, as they're not specific to PyOTA (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

BTW, nice to see another PyCharm user!